test-plugins/org.yakindu.sct.generator.c.test/gtests/InEventQueueTest/InEventQueueTest.cc

```diff
@@ -37,7 +37,9 @@ void inEventQueueIface_askOracle(InEventQueue* handle) {
 
 
 TEST(InEventQueue, InEventQueue) {
-	inEventQueue_init(&machine);
+	ineventqueue_event internal_queue[20];
```

Comment:
This tests : in queue / internal queue / user allocation
There should be additional tests:
* in queue / internal queue / without user allocation
* in queue / without user allocation
* internal queue / without user allocation
* in queue / with user allocation
* internal queue / with user allocation